.ci/test.sh

```diff
@@ -4,6 +4,7 @@
 set -e
 
 if [[ "$TASK" == "rpkg" ]]; then
+  Rcript .ci/lint_r_code.R $(pwd)
```

Comment:
I think you meant "Rcrypt" ☠️ 
```suggestion
  Rscript .ci/lint_r_code.R $(pwd)
```

Reply:
ha yup that would do it